README.rst

```diff
@@ -37,8 +37,39 @@ This will create a butler repository at ``DATA/``, ingest the raw data into ``HS
 It will also run various checks of the data integrity of the processed output.
 The resulting repository in ``DATA/`` will take up about 18GB.
 
+Debugging ``HSC/runs/ci_hsc``
+-----------------------------
+
+If the run fails on a given task for the primary ``HSC/runs/ci_hsc`` run, it is easy to rerun just that task (after making appropriate fixes or debugging output).
```

Comment:
I would remove "it is easy". Instead, maybe "... run, you can run a specific task in your failed run with the following command, replacing `taskLabelToRerun` with the appropriate label from the DPR pipeline:"